File: Trimestre3-20250507T052143Z-001/Trimestre3/PHP/ejemplosAñoPasado/objetos/persona.php
<?php
class Persona {
    public $nombre;
    protected $edad;
    public function __construct($n, $edad){
        $this -> nombre = $n;
        $this -> edad = $edad;
    }
    public function saludo(){
        return "Hola, me llamo $this->nombre y tengo $this->edad años";
    }
}

File: Examenes/Examen3/Persona.cs
namespace facilitado;

public class Persona {
  private String nombre;
  private String apellidos;
  private int edad;
  public Persona (String nombre, String apellidos, int edad) {
      this.nombre = nombre;
      this.apellidos = apellidos;
      this.edad = edad;
  }
  protected String getNombre () {
    return apellidos + ", " + nombre; }
  protected int getEdad () {
    return edad; }
} 